src/_includes/partials/links_list.liquid

```diff
@@ -8,7 +8,7 @@
     <li class="group">
       <a
         class="group-odd:bg-blue-50 group-odd:bg-opacity-50 p-4 pr-16 sm:p-6 md:p-8 sm:text-lg md:text-xl grid grid-cols-1 md:grid-cols-[1fr,2fr] lg:grid-cols-[2fr,3fr,1fr] gap-2 sm:gap-8 transition group-hover:text-magenta-500 relative"
-        href="{{ link.url }}"
+        href="{{ link.link }}"
```

Comment:
Good catch. This was not a smart naming strategy from my side. Feel free to improve this, e.g. by actually changing the fields in `further_reading` to `url` or the variable `link` to `resource`